config/karma-travis.conf.js

```diff
@@ -0,0 +1,28 @@
+const baseKarma = require('./karma.conf.js');
+process.env.CHROME_BIN = require('puppeteer').executablePath();
```

Comment:
Why even use `puppeteer` if the chrome binaries are already installed through Travis? We should either remove pupeteer, or remove the `chrome addon` of Travis.

Reply:
Removing Puppeteer did indeed work. Thanks for the feedback!